v1.7/ics/README.md

```diff
@@ -0,0 +1,12 @@
+To reproduce:
+
+```
+% bx cs cluster-create --name conformance
+% $(bx cs cluster-config conformance | grep export)
+% curl -L https://raw.githubusercontent.com/cncf/k8s-conformance/master/sonobuoy-conformance-1.7.yaml | kubectl apply -f -
+% kubectl logs -f -n sonobuoy sonobuoy
+
+# wait for "no-exit was specified, sonobuoy is now blocking"
+
+% kubectl cp sonobuoy/sonobuoy:/tmp/sonobuoy ./results
```

Comment:
I believe you need to drop the `/` in `/tmp/sonobuoy`.  At least, I did when I ran.

Reply:
Yeah, I need to rerun, i think the paths changed in a recent conformance update.